### DISCUSSION TITLE
Chrome and Firefox Browser Extension Usage

### DISCUSSION THREAD

**Discussion by BrittoZlink** [hoppscotch/hoppscotch#2476] [Q&A] [answered]
I am new to Hoppscotch, can any one please help me how to use the browser extension to send request? I found no help notes about how to use browser extension. please help me.

**Comment by liyasthomas** [comment, marked as answer]
Hi @BrittoZlink, our browser extension helps to fix CORS issues while testing APIs served on localhost.

Refer to this discussion thread answer for more info: https://github.com/hoppscotch/hoppscotch/discussions/2051#discussioncomment-1887545

**Reply by BrittoZlink** [reply]
Thank you bro...
